workflow-examples/src/test/java/com/redhat/parodos/examples/integration/KubeapiWorkFlow.java

```diff
@@ -0,0 +1,130 @@
+package com.redhat.parodos.examples.integration;
+
+import com.redhat.parodos.sdk.api.ProjectApi;
+import com.redhat.parodos.sdk.api.WorkflowApi;
+import com.redhat.parodos.sdk.api.WorkflowDefinitionApi;
+import com.redhat.parodos.sdk.invoker.ApiClient;
+import com.redhat.parodos.sdk.invoker.ApiException;
+import com.redhat.parodos.sdk.invoker.Configuration;
+import com.redhat.parodos.sdk.model.*;
+import com.redhat.parodos.workflow.utils.CredUtils;
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+import static org.junit.Assert.fail;
+
+public class KubeapiWorkFlow {
+
+	private final String projectName = "project-1";
+
+	private final String projectDescription = "Kubeapi example project";
+
+	private final String workflowName = "kubeapiWorkFlow";
+
+	private final String taskName = "kubeapiTask";
+
+	@Test
+	public void runFlow() throws ApiException {
+		ApiClient defaultClient = Configuration.getDefaultApiClient();
+
+		defaultClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds("test", "test"));
+
+		ProjectApi apiInstance = new ProjectApi(defaultClient);
+
+		try {
+			// RETRIEVE ALL PROJECTS AVAILABLE
+			List<ProjectResponseDTO> projects = apiInstance.getProjects();
+
+			// CHECK IF project ALREADY EXISTS
+			ProjectResponseDTO testProject = projects.stream()
+					.filter(prj -> projectName.equals(prj.getName()) && projectDescription.equals(prj.getDescription()))
+					.findAny().orElse(null);
+
+			// CREATE PROJECT IF NOT EXISTS
+			if (testProject == null) {
+				// DEFINE A PROJECT REQUEST
+				ProjectRequestDTO projectRequestDTO = ProjectRequestDTO.builder().name(projectName)
+						.description(projectDescription).build();
+
+				testProject = apiInstance.createProject(projectRequestDTO);
+			}
+
+			// ASSERT PROJECT IS PRESENT
+			projects = apiInstance.getProjects();
+			assertNotNull(projects.stream()
+					.filter(prj -> projectName.equals(prj.getName()) && projectDescription.equals(prj.getDescription()))
+					.findAny().orElse(null));
+
+			// GET workflow DEFINITIONS
+			WorkflowDefinitionApi workflowDefinitionApi = new WorkflowDefinitionApi();
```

Comment:
for consistency, either use `new WorkflowDefinitionApi(defaultClient)` or remove line 32 and update `new ProjectApi(defaultClient);` in line 36 to `new ProjectApi();`

Reply:
again, code copied from existing files

Reply:
again, `UseSDKExample` it's a dummy example

Reply:
will fix